Version 1.300 (minus final kerning)

dbenjaminmiller opened the pull request:
Per the FONTLOG:

- Add small caps ‘smcp’ for the regular text font.
- Add T and S with comma below for setting Romanian.
- Add superior* and inferior figures.
- Add alternate old-style figures, accessible via ‘ss11’.
- Fix misplaced accents on ‘i’.*
- Add ‘ffj’ ligature.*
- Fix thickness of connector part in division symbol.*
_*Already merged_

Now, just to add kerning for some of the Latin-Extended characters.

khaledhosny commented on FONTLOG.txt:
```diff
@@ -34,6 +34,16 @@ http://github.com/alif-type/xits
+- Add alternate old-style figures, accessible via ‘ss11’.
+- Fix misplaced accents on ‘i’.
+- Add ’ffj’ ligature.
+- Fix thickness of connector part in division symbol.
```
I thought I fixed that one already.

dbenjaminmiller replied:
You did but it wasn't in fontlog as it was since the last release. So I added it to the notes for the forthcoming one

khaledhosny commented on sources/smallcaps.fea:
```diff
@@ -0,0 +1,309 @@
+lookup c2scCapitalstoSmallCapitalslookup1 {
```
Make this lookup block a feature block (`feature c2sc {`) and remove the feature blocks below.

khaledhosny commented on sources/smallcaps.fea:
```diff
@@ -0,0 +1,309 @@
+    sub \uni021B by \uni021B.sc ;
+} smcpLowercasetoSmallCapitalslookup0;
+
+feature c2sc {
```
Removes these feature blocks. The languages and scripts are declared globally to ensure consistency. Inconsistencies in these can mean some lookups are not applied for some languages.

dbenjaminmiller commented:
Fixed the issue with `smallcaps.fea`. The FONTLOG didn't previously include the changes in the upcoming version, so I had added them — it still is not quite final, as it should include the forthcoming kerning addition (once you or I apply it) and the ultimate date etc. when the release is finalized.

khaledhosny commented:
> The FONTLOG didn't previously include the changes in the upcoming version,

Yes, I write them once when I’m making the release by checking the git log since the previous release tag.

dbenjaminmiller commented:
I can walk it back for now, then.

khaledhosny commented:
I squashed your commits into one and pushed a few cleanups and fixes. Hopefully I didn’t break anything.

dbenjaminmiller commented:
Looks good.

dbenjaminmiller commented:
CI build failed. It worked locally, no? Double check that, although the changes seemed fine

khaledhosny commented:
That is because I force-pushed before the build started.
